stdlib/mmap.pyi

```diff
@@ -68,7 +68,8 @@ class mmap(Iterable[int], Sized):
     @overload
     def __setitem__(self, __index: slice, __object: ReadableBuffer) -> None: ...
     # Doesn't actually exist, but the object is actually iterable because it has __getitem__ and
-    # __len__, so we claim that there is also an __iter__ to help type checkers.
+    # __len__, so we claim that there is also an __iter__ and __contains__ to help type checkers.
```

Comment:
I think `__contains__` deserves a separate comment, since it's a separate and independent issue. The problem is that `in` works on objects that define `__getitem__` but not `__contains__`, but type checkers (at least mypy and pyright; I just added support for this pattern in pyanalyze in quora/pyanalyze#564) don't know about this. That's a similar but separate issue to how objects with `__getitem__` are iterable.

I think this is also why the `SupportsGetItem` protocol has `__contains__`.